go/otgrpc/server.go

```diff
@@ -65,11 +61,80 @@ func OpenTracingServerInterceptor(tracer opentracing.Tracer, optFuncs ...Option)
 			}
 		} else {
 			ext.Error.Set(serverSpan, true)
-			serverSpan.LogFields(log.String("event", "gRPC error"), log.Error(err))
+			serverSpan.LogFields(log.String("event", "error"), log.String("message", err.Error()))
 		}
 		if otgrpcOpts.decorator != nil {
 			otgrpcOpts.decorator(serverSpan, info.FullMethod, req, resp, err)
 		}
 		return resp, err
 	}
 }
+
+// OpenTracingStreamServerInterceptor returns a grpc.StreamServerInterceptor suitable
```

Comment:
(same comment about clarifying how many Spans will be used to model the streaming RPC)